src/main/resources/org/mal_lang/langspec/lang.schema.json

```diff
@@ -0,0 +1,554 @@
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/steps"
+            }
+          ]
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "meta",
+        "type",
+        "tags",
+        "risk",
+        "ttc",
+        "requires",
+        "reaches"
+      ]
+    },
+    "risk": {
+      "type": "object",
+      "properties": {
+        "isConfidentiality": {
+          "type": "boolean"
+        },
+        "isIntegrity": {
+          "type": "boolean"
+        },
+        "isAvailability": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "isConfidentiality",
+        "isIntegrity",
+        "isAvailability"
+      ]
+    },
+    "steps": {
+      "type": "object",
+      "properties": {
+        "overrides": {
+          "type": "boolean"
+        },
+        "stepExpressions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/stepExpression"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "overrides",
+        "stepExpressions"
+      ]
+    },
+    "ttcExpression": {
+      "allOf": [
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "enum": [
+                "addition",
+                "subtraction",
+                "multiplication",
+                "division",
+                "exponentiation",
+                "function",
+                "number"
+              ]
+            }
+          },
+          "required": [
+            "type"
+          ]
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "enum": [
+                  "addition",
+                  "subtraction",
+                  "multiplication",
+                  "division",
+                  "exponentiation"
+                ]
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "type": true,
+              "lhs": {
+                "$ref": "#/definitions/ttcExpression"
+              },
+              "rhs": {
+                "$ref": "#/definitions/ttcExpression"
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "lhs",
+              "rhs"
+            ]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "function"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "type": true,
+              "name": {
+                "$ref": "#/definitions/identifier"
+              },
+              "arguments": {
+                "type": "array",
+                "items": {
+                  "type": "number"
```

Comment:
Do we want a number definition with min and/or max values?

Reply:
arguments to ttc distributions can be any finite number, so adding a limit would arbitrarily limit what can be done